webapp/src/pages/Game.jsx

```diff
@@ -4,45 +4,40 @@ import { Center } from "@chakra-ui/layout";
 import { useNavigate } from "react-router-dom";
 import Confetti from "react-confetti";
 import ButtonEf from '../components/ButtonEf';
-import {getQuestion} from '../components/game/Questions';
+import {getQuestion, answerQuestion} from '../components/game/Questions';
 
 export default function Game() {
 	const navigate = useNavigate();
 
-	const [question, setQuestion] = useState({ id:1, content: "default question", answers: [], questionCategory: "", answerCategory: "", language: "en", type: ""});
-
-	// const generateQuestion = () => {
-	// 	const fetchQuestion = async () => {
-	// 		const result = await getQuestion();
-	// 		setQuestion(result);
-	// 	};
-	// }
-	// componentDidMount() {
-	// 	generateQuestion();
-	// }
-
+	const [question, setQuestion] = useState({ id:1, content: "default question", answers: [{id:1, text:"answer1", category:"category1" }, {id:2, text:"answer2", category:"category2" }], questionCategory: "", answerCategory: "", language: "en", type: ""});
 	useEffect(() => {
 		const fetchQuestion = async () => {
-		  const result = await getQuestion();
-		  setQuestion(result);
+		  await generateQuestion();
 		};
 		fetchQuestion();
-	  }, []);
+	}, []);
+
+	const generateQuestion = async () => {
+		const result = await getQuestion();
+		setQuestion(result);
+	};
 
+	const [answer, setAnswer] = useState({id:1, text:"answer1", category:"category1" });
 	const [selectedOption, setSelectedOption] = useState(null);
 	const [nextDisabled, setNextDisabled] = useState(true);
 	const [roundNumber, setRoundNumber] = useState(1);
 	const [correctAnswers, setCorrectAnswers] = useState(0);
 	const [showConfetti, setShowConfetti] = useState(false);
 
 	const answerButtonClick = (option) => {
+		setAnswer(question.answers[option]);
```

Comment:
ejem you need to do (option-1) since options start at 1 and end at 2 so it will give an index out of bounds error :)